Guide/tailwindcss.markdown

```diff
@@ -6,52 +6,64 @@
 
 ## Introduction
 
-Yes, while bootstrap is the default CSS framework in IHP, you can use IHP together with TailwindCSS (TW in short). This guide will help you set up the latest Tailwind version in your IHP project. We will be leveraging the TW's [JIT](https://tailwindcss.com/docs/just-in-time-mode) mode.
-
-We will also have PostCSS added as part of the installation. PostCSS is used for nesting CSS, minifying, striping out comments, etc.
+Yes, while bootstrap is the default CSS framework in IHP, you can also use IHP together with Tailwind CSS.
 
 ## Installing
 
-### Standalone CLI
-
-Tailwind offers a Standalone CLI for frameworks like IHP, Rails and Phoenix as an alternative to installing npm just for the sake of Tailwind.
-
-You can follow the [official article](https://tailwindcss.com/blog/standalone-cli) for checking a standalone Tailwind binary into your project.
-
-If you require cross-platform support, [ihp-tailwind-bootstrapper](https://github.com/ship-nix/ihp-tailwind-bootstrapper) uses Tailwind's official binaries and let's you automatically use the correct one for your system. Only the binary used for deployment is checked into version control.
-
-
-### NodeJS
+### Adding package to flake.nix
 
-First, we need to add NodeJS to our project. **You should also follow this step if you have NodeJS already installed on your system.** Installing the NodeJS version via nix allows all people working on your project to get the same NodeJS version as you're using.
-
-For that open your projects `default.nix` and add `nodejs` to `otherDeps`:
+In the `flake.nix`, add the `tailwindcss` package bundled with the most common official plugins.
 
 ```nix
-otherDeps = p: with p; [
+...
+packages = with pkgs; [
     # Native dependencies, e.g. imagemagick
-    nodejs
+    (nodePackages.tailwindcss.overrideAttrs
+        (oa: {
+            plugins = [
+                nodePackages."@tailwindcss/aspect-ratio"
+                nodePackages."@tailwindcss/forms"
+                nodePackages."@tailwindcss/language-server"
+                nodePackages."@tailwindcss/line-clamp"
+                nodePackages."@tailwindcss/typography"
+            ];
+        })
+    )
 ];
+...
 ```
 
-Now you need to rebuild your local development environment:
+Rebuild your development environment to fetch the added package:
 
 ```bash
-devenv up
+nix flake update
 ```
 
-After that, you have `node` and `npm` available in your project.
+After that, you should be able to verify that `tailwindcss` CLI is available in your project directory.
 
-### Installing Tailwind
+```
+$ tailwindcss
+tailwindcss v3.2.7
```

Comment:
This is what I got

```suggestion
tailwindcss v3.3.3
```

Reply:
I am guessing you are running on master, and I am running on v1.1.0.

Maybe not necessary to have any of the output there